src/ui-test/tests/camel.settings.test.ts

```diff
@@ -125,6 +127,34 @@ describe('Camel User Settings', function () {
 
     });
 
+    describe('Update Extra Launch Parameter', function () {
+
+        const customExtraLaunchParameter = '--fresh';
+
+        beforeEach(async function () {
+            await prepareEnvironment();
+        });
+
+        afterEach(async function () {
+            await cleanEnvironment();
+            resetUserSettings(EXTRA_LAUNCH_PARAMETER_ID);
+        });
+
+        it('Should use default extra launch parameter', async function () {
+            await executeCommand(CAMEL_RUN_ACTION_QUICKPICKS_LABEL);
+
+            await waitUntilTerminalHasText(driver, [`${defaultExtraLaunchParameter}`], 6000, 120000);
+        });
+
+        it(`Should use user defined extra launch parameter'${customExtraLaunchParameter}'`, async function () {
+            await setJBangVersion(customExtraLaunchParameter);
```

Comment:
I think the other tests are failing because the Camel version setting is modified to --fresh and not set back

Reply:
- The tests are failing only for vscode version 1.82.3. I tested locally with this version and they run without errors:

```
$ CODE_VERSION=1.82.3 npm run ui-test

> vscode-debug-adapter-apache-camel@0.13.0 preui-test
> npm run compile


> vscode-debug-adapter-apache-camel@0.13.0 compile
> tsc -p ./ && npm run lint


> vscode-debug-adapter-apache-camel@0.13.0 lint
> eslint src --ext ts

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.7.4 <5.4.0

YOUR TYPESCRIPT VERSION: 5.4.3

Please only submit bug reports when using the officially supported version.

=============

> vscode-debug-adapter-apache-camel@0.13.0 ui-test
> node --unhandled-rejections=warn-with-error-code out/ui-test/uitest_runner.js



WARNING: You are using the outdated VS Code version '1.82.3'. The latest stable version is '1.88.0'.


Downloading VS Code: 1.82.3 / stable
VS Code exists in local cache, skipping download
ChromeDriver 114.0.5735.90 exists in local cache, skipping download
Executing prepublish script 'npm run vscode:prepublish'...

> vscode-debug-adapter-apache-camel@0.13.0 vscode:prepublish
> npm run compile


> vscode-debug-adapter-apache-camel@0.13.0 compile
> tsc -p ./ && npm run lint


> vscode-debug-adapter-apache-camel@0.13.0 lint
> eslint src --ext ts

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.7.4 <5.4.0

YOUR TYPESCRIPT VERSION: 5.4.3

Please only submit bug reports when using the officially supported version.

=============
This extension consists of 849 files, out of which 591 are JavaScript files. For performance reasons, you should bundle your extension: https://aka.ms/vscode-bundle-extension . You should also exclude unnecessary files by adding them to your .vscodeignore: https://aka.ms/vscode-vscodeignore
 DONE  Packaged: /home/mdinizde/Development/Projects/camel-dap-client-vscode/vscode-debug-adapter-apache-camel-0.13.0.vsix (849 files, 5.92MB)
Installing extensions...
(node:1359519) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `code --trace-deprecation ...` to show where the warning was created)
Extension 'vscode-debug-adapter-apache-camel-0.13.0.vsix' was successfully installed.


Detected user defined code settings
Writing code settings to /tmp/test-resources/settings/User/settings.json
Launching browser...
Browser ready in 4185 ms
Launching tests...
  Camel User Settings
    Update Camel Version
      ✔ Should use '3.20.1' user defined Camel version (24207ms)
    Update JBang Version
      ✔ Should use default JBang version (9669ms)
      ✔ Should use user defined JBang version '3.20.5' (15272ms)
    Update Maven Repository
      ✔ Should use '3.20.1.redhat-00026' user defined Camel Version and Red Hat Maven Repository (15173ms)
      ✔ Should not use '#repos' placeholder for global Camel JBang repository config (20615ms)
    Update Extra Launch Parameter
      ✔ Should use default extra launch parameter (9559ms)
      ✔ Should use user defined extra launch parameter'--fresh' (15244ms)

Shutting down the browser
Uninstalling redhat.vscode-debug-adapter-apache-camel...
Extension 'redhat.vscode-debug-adapter-apache-camel' was successfully uninstalled!

  7 passing (3m)
```

- By looking at the screenshots of the failing tests for the windows build it seems that the test isnot able to retrieve the JBang version property in the settings page.
- For the ubuntu build I could not find anything relevant in the screenshots.